app/src/androidTest/java/com/conupods/IndoorMaps/ConcreteBuildingTest.java

```diff
@@ -0,0 +1,78 @@
+package com.conupods.IndoorMaps;
+
+import com.conupods.IndoorMaps.ConcreteBuildings.CCBuilding;
+import com.conupods.IndoorMaps.ConcreteBuildings.VLBuilding;
+import com.conupods.IndoorMaps.ConcreteBuildings.MBBuilding;
+import com.conupods.IndoorMaps.ConcreteBuildings.HBuilding;
+import com.conupods.OutdoorMaps.Models.Building.Building;
+import com.google.android.gms.maps.model.LatLng;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConcreteBuildingTest {
+
+    @Test
+    public void CCBuilding(){
+
+        Building CC = CCBuilding.getInstance();
+
+        assertEquals("7141 Sherbrooke West",CC.getAddress());
+        assertEquals("CC",CC.getCode());
+        assertEquals("LOY",CC.getConcreteParent());
+        assertEquals("Central Building",CC.getLongName());
+        assertEquals("CC Building",CC.getName());
+        assertEquals(new ArrayList<>(),CC.getClassRooms());
+        assertEquals(new LatLng(45.458204,-73.6403),CC.getLatLng());
```

Comment:
the coordinates should be declared in variables for readability and documentation purposes